### DISCUSSION TITLE
My Strategy which yields 100% winrate

### DISCUSSION THREAD

**Discussion by Sumanth-R-Kovalli** [EA31337/EA31337-strategies#171] [Strategy Ideas]
I have  made one strategy which yields  100% winrate ,but I don't know to write the code for my strategy ,but the orders placed for my strategy cannot be placed by humans (it can be but it takes time ) if anyone want to hear my strategy and write code for it.....?

**Comment by activepc** [comment]
Hi 

 

You can send it to me !   

 

I don’t think there exist a strategy that is 100%, but maybe you got it !  

 

Thanks

Pieter.

 

From: SUMANTH R K ***@***.***> 
Sent: Wednesday, 27 November 2024 12:23
To: EA31337/EA31337 ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [EA31337] My Strategy (Discussion EA31337/EA31337#429)

 

I have made one strategy which yields 100% winrate ,but I don't know to write the code for my strategy ,but the orders placed for my strategy cannot be placed by humans (it can be but it takes time ) if anyone want to hear my strategy and write code for it.....?

**Comment by kenorb** [comment]
You can describe the strategy here.
